src/docs/cookbook/testing/integration/introduction.md

```diff
@@ -247,23 +249,60 @@ void main() {
 
 ### 6. Run the tests
 
-Now that you have an instrumented app and a test suite, run the tests.
-First, be sure to launch an Android Emulator, iOS Simulator,
+Now that you have an instrumented app _and_ a test suite,
+run the tests. The process of running the integration
+tests varies depending on the platform you are testing
+against. You can test against a mobile platform or the web.
+
+#### 6a. Mobile
+
+To test on iOS or Android,
+launch an Android Emulator, iOS Simulator,
 or connect your computer to a real iOS / Android device.
 
 Then, run the following command from the root of the project:
 
-```
+```shell
 flutter drive --target=test_driver/app.dart
 ```
 
-This command:
+This command performs the following:
+
+* Builds the `--target` app and installs
+  it on the emulator / device.
+* Launches the app.
+* Runs the `app_test.dart` test suite located
+  in `test_driver/` folder.
+
+---
+
+#### 6b. Web
+
+To test for web,
+determine which browser you want to test against
+and download the corresponding web driver:
+
+  * Chrome: [Download ChromeDriver][]
+  * Firefox: [Download GeckoDriver][]
+  * Safari: Safari can only be tested on a Mac;
+    the SafariDriver is already installed on Mac machines.
+  * Edge [Download EdgeDriver][]
 
-  1. Builds the `--target` app and installs it on the emulator / device.
-  2. Launches the app.
-  3. Runs the `app_test.dart` test suite located in `test_driver/` folder.
+Launch the WebDriver, for example: 
 
+```terminal
+./chromedriver --port=4444
+```
+From the root of the project,
+run the following command:
+
+```terminal
+flutter drive --target=test_driver/[test_app_file].dart --browser-name=[browser name] --release
```

Comment:
Can you change "[test_app_file]" to "app" to be consistent with the command for Mobile.